flake.nix

```diff
@@ -56,7 +56,7 @@
           })
```

Comment:
Re. the naming for let bindings, is there a standard somewhere that camel case is preferred? Or maybe just that camel > kebab > snake? I see all kinds of stuff. Maybe we could setup a lint if there is a standard somewhere

Reply:
https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md says use camel case for (non-package-name) variable names.